File: fix_sales_modal.py
import re

with open('src/app/admin/orders/page.tsx', 'r', encoding='utf-8') as f:
 content = f.read()

# Fix: backslash-backtick -> backtick
content = content.replace(chr(92) + chr(96), chr(96))
# Fix: backslash-dollar-brace -> dollar-brace in DS.border context
content = content.replace(chr(92) + '${DS.border}', '${DS.border}')

with open('src/app/admin/orders/page.tsx', 'w', encoding='utf-8') as f:
 f.write(content)

print('Done')

# Verify
with open('src/app/admin/orders/page.tsx', 'r', encoding='utf-8') as f:
 lines = f.readlines()

print('Line 900:', repr(lines[899]))
print('Line 919:', repr(lines[918]))

File: fix_sales_modal.js
const fs = require('fs');
let c = fs.readFileSync('src/app/admin/orders/page.tsx', 'utf8');
// Char codes: 92 = backslash, 96 = backtick
const bs = String.fromCharCode(92);
const bt = String.fromCharCode(96);
// Fix backslash-backtick -> backtick
c = c.split(bs + bt).join(bt);
// Fix backslash-dollar -> dollar in DS.border context
c = c.split(bs + '${DS.border}').join('${DS.border}');
fs.writeFileSync('src/app/admin/orders/page.tsx', c);
console.log('Fixed');
const lines = c.split('\n');
console.log('Line 900:', JSON.stringify(lines[899]));
console.log('Line 919:', JSON.stringify(lines[918]));
